BeamDet/ERBeamDetDigitizer.cxx

```diff
@@ -83,13 +83,13 @@ InitStatus ERBeamDetDigitizer::Init() {
     Fatal("Init", "Can`t find collection BeamDetMWPCPoint!"); 
 
   // Register output array fRTelescopeHits
-  fBeamDetToFDigi1 = new TClonesArray("ERBeamDetTOFDigi",1000);
-  fBeamDetToFDigi2 = new TClonesArray("ERBeamDetTOFDigi",1000);
+  fBeamDetToFDigi1 = new TClonesArray("ERDigi",1000);
+  fBeamDetToFDigi2 = new TClonesArray("ERDigi",1000);
```

Comment:
Here and everywhere TClones array must be sized according to  the detector's channels number for digies, tracks, PIDs